META.json

```diff
@@ -1,24 +1,24 @@
 {
    "name": "ApacheAGE",
-   "abstract": "Apache AGE is a PostgreSQL Extension that provides graph database functionality",
-   "description": "Apache AGE is a PostgreSQL Extension that provides graph database functionality. AGE is an acronym for A Graph Extension, and is inspired by Bitnine's fork of PostgreSQL 10, AgensGraph, which is a multi-model database. The goal of the project is to create single storage that can handle both relational and graph model data so that users can use standard ANSI SQL along with openCypher, the Graph query language. A graph consists of a set of vertices (also called nodes) and edges, where each individual vertex and edge possesses a map of properties. A vertex is the basic object of a graph, that can exist independently of everything else in the graph. An edge creates a directed connection between two vertices. A graph database is simply composed of vertices and edges. This type of database is useful when the meaning is in the relationships between the data. Relational databases can easily handle direct relationships, but indirect relationships are more difficult to deal with in relational databases. A graph database stores relationship information as a first-class entity. Apache AGE gives you the best of both worlds, simultaneously.",
-   "version": "1.3.0",
+   "abstract": "Graph database functionality for PostgreSQL",
+   "description": "Apache AGE is a PostgreSQL Extension that provides graph database functionality. The goal of the project is to create single storage that can handle both relational and graph model data so that users can use standard ANSI SQL along with openCypher, the Graph query language.",
+   "version": "1.5.0",
    "maintainer": [
       "users@age.apache.org"
    ],
    "license": "apache_2_0",
    "provides": {
-      "ApacheAGE": {
+      "age": {
          "abstract": "Apache AGE is a PostgreSQL Extension that provides graph database functionality",
-         "file": "age--1.3.0.sql",
+         "file": "age--1.5.0.sql",
          "docfile": "README.md",
-         "version": "1.3.0"
+         "version": "1.5.0"
       }
    },
    "prereqs": {
       "runtime": {
          "requires": {
-            "PostgreSQL": "14.0.0"
+            "PostgreSQL": "11.0.0"
```

Comment:
Is this the minimum PG? If so, while we do support 11, we've stopped most new additions to 11 & 12. 13 would be better.

Reply:
Additionally, we support up to PG16.

Reply:
Yes, this is the minimum supported version. I took it from the README, which says:

> for now AGE supports Postgres 11, 12, 13, 14, 15 & 16.

Do you want to drop support for 11 and 12?

Reply:
Would this cause it to build against 11? The main issue with 11 & 12 is that we have mostly dropped support of them beyond 1.5.0.

Reply:
Btw, a lot of this has to do with 11 and 12 being much different from 13 and above.

Reply:
It just means that it supports PostgreSQL 11 and up. If someone downloaded it and looked, they'd know they could build it against Postgres 11. At some point a smoke test setup might depend on this config to decide what versions of Postgres to test builds against.

Reply:
@rafsun42 @MuhammadTahaNaveed @dehowef @Zainab-Saad 
thoughts?

Reply:
@theory An age release for PG16 (which is master branch currently) will not compile on Postgres 11, or any other Postgres version (at least it is not meant to). For context, we have different development branches for different PG versions. Given that, I believe the version in the meta file would depend on the branch this PR is being applied on. For example, if the PR is being applied on PG15, then the version can point to the latest minor version of PG15.

Reply:
Also, be aware that these will be considered separate distributions by PGXN. Wheres today you have `https://pgxn.org/dist/apacheage/`, future releases will be under `https://pgxn.org/dist/apacheage16/`, `https://pgxn.org/dist/apacheage15/`, etc.